src/minuteman_nfq.erl

```diff
@@ -212,12 +212,12 @@ build_send_cfg_msg(Socket, Command, Queue, Pf) ->
 
 nfnl_query(Socket, Query) ->
   Request = netlink:nl_ct_enc(Query),
+  lager:debug("Request: ~p", [Request]),
   gen_socket:sendto(Socket, netlink:sockaddr_nl(netlink, 0, 0), Request),
   Answer = gen_socket:recv(Socket, 8192),
-  ?MM_LOG("Answer: ~p~n", [Answer]),
+  lager:debug("Answer: ~p", [Answer]),
   case Answer of
     {ok, Reply} ->
-      ?MM_LOG("Reply: ~p~n", [netlink:nl_ct_dec(Reply)]),
```

Comment:
I was thinking that instead of deleting it, we could just do `netlink:nl_ct_dec(Reply)` in a separate function call can use the result in the log as well as the case statement